dotcom-rendering/src/web/components/Branding.importable.tsx

```diff
@@ -33,11 +30,28 @@ const brandingAboutLink = (palette: Palette) => css`
 	}
 `;
 
-export const Branding: React.FC<{
-	branding: Branding;
+const getDimensions = ({
+	logo: { dimensions },
+}: Branding): { width: number; height: number } => {
+	if (dimensions.width <= 140) return dimensions;
+	const scaling = 140 / dimensions.width;
+	return {
+		width: 140,
+		height: Math.round(dimensions.height * scaling),
+	};
+};
```

Comment:
🤔 Having javascript just for a logo feels wrong to me. I can see that we enabled the execution of javascript on this image so that we can track clicks. I think based on some recent conversations and where our priorities are going we might want to revisit that and maybe there are better ways to do this tracking without the need for another island.

This is in theory irrelevant to your change but I see we are now adding more javascript which would make future simplifications harder to do. Is there a way to fix this width problem using just css?

Reply:
Oh no, I think it’s absolutely bonkers to load react to track a single click! But that’s for another PR :)

Reply:
Wholehearteadly agree, I had difficulty looking the other way as this is so wasteful! This is for another PR, however.

I would rather we inlined the function run in `onClick` rather than hydrate this part of the DOM with Preact and run some Emotion. Making this an Island is objectively the wrong approach.

Reply:
This does not make future refactoring any harder, as this component shold be server rendered. As you can see in the bundle analyser, we’re actually getting a smaller island with this code, because Preact + Emotion + Source foundations are actually a lot larger than my addition!

Reply:
> 🤔 Having javascript just for a logo feels wrong to me.

By javascript - do you mean client-side javascript? From what I understand in this PR is this JS is only run on the server.

If we do keep this method, could we call it something like `limitWidthToMax140` or something describing what it does?

Reply:
An enhancer 😻 !

Reply:
+1 to inlining (as a separate PR)